f8a_worker/workers/cvedbsync.py

```diff
@@ -28,7 +29,9 @@ def _get_snyk_vulndb(self):
             # clone vulndb git repo
             self.log.debug("Cloning snyk/vulndb repo")
             # 'develop' branch seems to be a bit more stable than master
-            Git.clone(self._VULNDB_GIT_REPO, vulndb_dir, depth="1", branch="develop")
+            git = Git.clone(self._VULNDB_GIT_REPO, vulndb_dir, depth="1", branch="develop")
+            if git is None:
+                raise FatalTaskError("Couldn't clone %s" % self._VULNDB_GIT_REPO)
```

Comment:
Maybe we could move this check directly to `Git.clone()` so we don't check for clone success each time and unify git clone error reporting.